source/integrate/community-for-mattermost-for-microsoft-teams.rst

```diff
@@ -0,0 +1,71 @@
+Community for Mattermost for Microsoft 365
+==========================================
+
+.. include:: ../_static/badges/ent-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+Connect with the Community for Mattermost for seamless collaboration.
+
+The Community for Mattermost app for Microsoft 365 provides seamless access to the public Mattermost Community instance from inside the tools you use every day. Join thousands of Mattermost users, contributors, and staff members in a vibrant community where you can ask questions, get support, share ideas, and contribute to shaping the future of Mattermost.
+
+.. image:: ../images/mattermost_community_for_microsoft_365.png
+  :alt: Community for Mattermost for Microsoft 365, available in the Microsoft App Store.
+
+Key Features
+------------
+
+- **Direct Access**: Access the Community for Mattermost directly from a tab without switching applications or opening a browser
+- **Seamless Integration**: Experience the full functionality of Community for Mattermost within a familiar interface
+- **Real-time Collaboration**: Engage with the Community for Mattermost in real-time discussions on product features, technical questions, and best practices
+- **Product Support**: Get help from both Mattermost staff and experienced community members
+- **Contribute to Development**: Participate in discussions that shape the future direction of Mattermost products
+- **Knowledge Sharing**: Learn implementation strategies and best practices from a diverse community of users
+- **Stay Updated**: Keep up with the latest Mattermost announcements, updates, and roadmap information
+
+This app is designed to work with Microsoft 365, Outlook and Microsoft Teams.
+
+Setup
+-----
+
+Install the Community for Mattermost for Microsoft 365 application
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  
+
+Any Microsoft 365 user can complete the steps below to access the Community for Mattermost within their Microsoft environment.
+
+1. Sign in to your Microsoft Teams account from a `browser <https://teams.microsoft.com/v2/?clientexperience=t2>`_ or the desktop application.
+
+2. Select the **[+] Apps** button in the Teams sidebar.
+
+3. Search for "Community for Mattermost" and then select **Add** to install the application.
+
+4. (Optional) Pin the application to your Teams sidebar by right-clicking on it and selecting **Pin**.
+
+5. Once the Community for Mattermost app is installed, you'll be connected to the public Mattermost Community instance.
```

Comment:
I'm personally accessing Hub, not Community. Intentional?

Reply:
See https://hub.mattermost.com/microsoft-teams/pl/btkiocy6kfy68cf5715h3epeur for a bit more context, but in short this is the "public app" we want to list in the global apps store. If customers want to point at their own app (i.e. how we have Hub setup), it requires more effort. The public app is intended to champion our ongoing renewal strategies.